### ISSUE TITLE
2 "Shared: Forward Rederer" in dropdown??

### ISSUE THREAD

**Issue by dfkeenan** [SiliconStudio/paradox#602]
See right side of image:

![image](https://user-images.githubusercontent.com/6070165/27253177-0a589374-53b3-11e7-9327-11bc13abe871.png)

**Comment by xen2** [comment]
It represents the two forward renderer instances that you have in your graph on the left.

UX is currently not great because they don't have name and there is no "highlight" visual cues to help you understand those are the instances of the graph on the left.

**Comment by xen2** [comment]
Note: current graphics compositor editor is in a "temporary transition state".
After some internal discussion we would like to do it quite differently but this will require quite a lot of work.

**Comment by dfkeenan** [comment]
OK, just a little confusing. So If I chose either to add another it wouldn't mater because it's the same instance?

While more powerful (fine grain) the new compositor is a little confusing. I was trying to work out how to add multiple layers. The old setup made that more obvious.

**Comment by jamesduffytokyo** [comment]
Yep, the graphics compositor is extremely confusing. Needs an overhauled UI one day.

**Comment by dfkeenan** [comment]
Yep, I still don't know what I am doing.  Was trying to workout how to accomplish the same thing as my answer to this question http://answers.xenko.com/questions/2301/ui-draw-order.html#answer-2302 in 2.X.  Not sure how far away your new version is or how much simpler it would be. But in the mean time could you please provide documentation on how to achieve:

* Layering
* Custom post processing (image) effects.

**Comment by jamesduffytokyo** [comment]
@dfkeenan is this useful? http://doc.xenko.com/latest/en/manual/graphics/post-effects/color-transforms/custom-color-transforms.html

**Comment by dfkeenan** [comment]
@jamesduffytokyo that definitely helps with custom post processing.  Though I am a little disappointed you have to jump through so many hoops to do custom post processing before the built in ones. But that's an API design decision.  I still have trouble with the shading language but that's a different topic. 

Documentation/example about layering render groups would be very helpful.  That might be a moving target for the "Composition Improvements".

FYI. the "Xenko shading language" link in the first point under section "1. Create a shader" is pointing to the color transforms page not the shader language page.

**Comment by jamesduffytokyo** [comment]
@dfkeenan Thanks for catching the bad link. Fixed.
